.gitignore

```diff
@@ -12,4 +12,5 @@ __pycache__/
 local_settings.py
 db.sqlite3
 
-tmp
\ No newline at end of file
+tmp
+Pipfile.lock
```

Comment:
@wmemorgan don't think we want to add the Pipfile.lock into .gitignore

Reply:
I didn't see a Pipfile.lock in the original repo so I thought we should keep it out. But I did some Googl-ing and it seems that is generally recommended to keep Pipfile.lock in version control despite having a different OS among team members. Here's the thread I found that discusses it:

https://github.com/pypa/pipenv/issues/598

Reply:
I see. Sounds good!